packages/html-reporter/src/reportView.tsx

```diff
@@ -85,19 +85,30 @@ export const ReportView: React.FC<{
         />
       </Route>
       <Route predicate={testCaseRoutePredicate}>
-        {!!report && <TestCaseViewLoader report={report} tests={filteredTests.tests} testIdToFileIdMap={testIdToFileIdMap} />}
+        {!!report ?
+          <TestCaseViewLoader report={report} tests={filteredTests.tests} testIdToFileIdMap={testIdToFileIdMap} /> :
+          <div className='error'>Report data could not be found</div>}
```

Comment:
What does this mean? What do I do?

Reply:
A fatal error occurred with loading the report and there's likely no recovery. The best thing the user can do is reload. We currently show a blank screen if this happens.

Reply:
How is `Report data could not be found` better than blank screen?

Reply:
Generally it is better to communicate to the user that something broke, rather than doing nothing. Technically a blank screen implicitly communicates something broke, but it's still a poor experience.

Any place where an error can occur but is not handled (generally explicitly) is a bug. This is no different. I agree that the error message could be better though.